src/backend/access/transam/xlog.c

```diff
@@ -7418,10 +7418,10 @@ CreateCheckPoint(int flags)
 	 */
 	SyncPreCheckpoint();
 
-	/*
-	 * NEON: perform checkpiont action requiring write to the WAL before we determine the REDO pointer.
-	 */
-	PreCheckPointGuts(flags);
+   /*
+    * NEON: perform checkpiont action requiring write to the WAL before we determine the REDO pointer.
+    */
+    PreCheckPointGuts(flags);
```

Comment:
unnecessary whitespace change. (You could fix the "checkpiont" typo while we're at it though)

Reply:
Fixed